index.d.ts

```diff
@@ -0,0 +1,42 @@
+import { Request, Response, NextFunction, Errback } from 'express';
+
+export enum CustomErrorTypes {
+  BAD_REQUEST= 'bad_request',
+  FORBIDDEN= 'forbidden',
+  NOT_FOUND= 'not_found',
+  SWAGGER_INPUT_VALIDATOR= 'swagger_input_validator',
+  SWAGGER_OUTPUT_VALIDATOR= 'swagger_output_validator',
+  SWAGGER_VALIDATOR= 'swagger_validator',
+  OAS_VALIDATOR= 'OpenAPIUtilsError:response',
+  UNAUTHORIZED= 'unauthorized',
+  WRONG_INPUT= 'wrong_input',
+}
+
+declare class CustomError {
+  constructor(type: CustomErrorTypes, message: any, extra: any);
+  name: CustomErrorTypes | string;
+  type: CustomErrorTypes | string;
+  message: any;
```

Comment:
everything looks 👌🏻 just one small suggestion for replace the any's with unknown because it offers type checking whenever we try to use the variable. 😃

Reply:
Done